apps/docs/src/migration.md

```
@@ -1,7 +1,27 @@
-All steps below are required unless otherwise stated. Please follow them in the correct
-order.
+All steps below are required unless otherwise stated. Directly upgrading across multiple
+major versions is not supported. If you want to upgrade from a version older than the last
+major release, please follow each major version's migration steps in order.
+
+## From `v9.*` to `v10.*`
+
+:::warning Environment Variables Change
+If you had `SCHEDULER_FREQUENT_CRON_JOBS_EVERY_MINUTES=2` in your environment, then change
+it to `SCHEDULER_FREQUENT_CRON_JOBS_SCHEDULE="every 2 minutes"`. Read more
+[here](./integrations/overview.md#yank-integrations).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)